app/containers/Forms/EmissionField.tsx

```diff
@@ -36,7 +36,6 @@ export const EmissionFieldComponent: React.FunctionComponent<FieldProps> = (
           .emission-toggle {
             text-align: right;
             position: relative;
```

Comment:
Is the `position:relative` still needed?

Reply:
Excellent point - no, I'm sure it's not. Can remove

Reply:
Yes please. I think it would be confusing to see this css rule which has no effect.